city_scrapers/spiders/wayne_state_university.py

```diff
@@ -0,0 +1,95 @@
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+import re
+
+
+class WayneStateUniversitySpider(CityScrapersSpider):
+    name = "wayne_state_university"
+    agency = "Wayne State University"
+    timezone = "America/Chicago"
+    allowed_domains = ["bog.wayne.edu"]
+    start_urls = ["http://bog.wayne.edu/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        
+        #one large all encompassing meeting
+        meeting_dates = response.xpath('//*[@id="content"]/div[1]/div/h2').re('((?:January|Feburary|March|April|May|June|July|August|October|September|November|December)\s+\d{1,2},\s+\d{4})')
```

Comment:
An easier way of pulling this would be to look at all of the `li` elements in the bottom section which you can get with:

```python
response.css(".px-4 ul > li")
```